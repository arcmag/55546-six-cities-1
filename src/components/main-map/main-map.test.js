import renderer from 'react-test-renderer';

import MainMap from './main-map';

const mock = {
  selectedCity: `Paris`,
  offers: [
    {
      title: `Beautiful & luxurious apartment at great location`,
      src: `img/apartment-01.jpg`,
      city: {
        name: `Paris`,
        coordinate: [48.852969, 2.351074]
      },
      price: 120,
      rating: 4.7,
      type: `Apartment`,
      coordinate: [48.852969123654, 2.351074654123],
      isChecked: false,
      isPremium: true
    },
    {
      title: `Beautiful & luxurious apartment at great location`,
      src: `img/apartment-01.jpg`,
      city: {
        name: `Paris`,
        coordinate: [48.852969, 2.351074]
      },
      price: 120,
      rating: 4.7,
      type: `Apartment`,
      coordinate: [48.852969123654, 2.351074654123],
      isChecked: false,
      isPremium: true
    }
  ]
};

describe(`Test MainMap`, () => {
  it(`test renderer`, () => {
    const tree = renderer.create(<MainMap
      offers={mock.offers}
      selectedCity={mock.selectedCity}
    />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
