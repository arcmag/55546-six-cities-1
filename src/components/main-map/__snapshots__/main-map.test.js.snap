// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test MainMap test renderer 1`] = `
<section
  className="cities__map map"
  id="map"
  style={
    Object {
      "height": "500px",
    }
  }
/>
`;
